client/src/components/content/BpmnModeler/BpmnModeler.jsx

```diff
@@ -75,9 +74,13 @@ const BpmnModeler = () => {
           <div className={styles['bpmn-modeler-container']} id='bpmnview' />
         </div>
       </Content>
-      <ModelerSidebar modeler={modeler} getRobotFile={downloadRobotFile} />
+      <ModelerSidebar robotId={robotId} modeler={modeler} getRobotFile={downloadRobotFile} />
```

Comment:
Do we need the displaying of the bot id in the sidebar or was this only for testing purposes?

Reply:
I think this is a good way to show that it works as long as the parsers aren't implemented and we can't look at the model to see if it works

Reply:
Longterm I would still put the robot name at least in the sidebar or somewhere else on the modeler page :)

Reply:
I agree with Daniel 100%. The `RobotId` is just to show that the right bot is taken and the `RobotId` is ready to use. In the long run, the robot name should be displayed there at the top (or at another place on the page) (unfortunately, I could not implement this as quickly as planned).